File: Python-Basics/cinema.py
movie_project = input()
rows = int(input())
column = int(input())

seats = rows * column
ticket = 0

if movie_project == "Premiere":
    ticket = 12
elif movie_project == "Normal":
    ticket = 7.50
elif movie_project == "Discount":
    ticket = 5
sales = ticket * seats
print(f"{sales:.2f} leva")

File: JS-Basics/Conditional Statements Advanced Exercise/cinema.js
function cinema(input){
    let movieType = input[0];
    let row = Number(input[1]);
    let column = Number(input[2]);
    let income = 0;

    if(movieType === "Premiere"){
        income = row * column * 12;
    }
    else if(movieType === "Normal"){
        income = row * column * 7.50;
    }
    else{
        income = row * column * 5;
    }

    console.log(income.toFixed(2) + " leva");
}
cinema(["Premiere",
"10",
"12"]);